### ISSUE TITLE
Should the school years be modeled as being a member of a group? Would it be the same for professors?

### ISSUE THREAD

**Issue by chin-rcip** [chin-rcip/collections-model#11]
### Modeling school years

The time someone is a student can be modeled in various different way, depending on how we see the education of someone.

In the v.1.5 of the Target Modem, education is modeled as a Social Role, where the university or school is merely a place, as shown in the following example with Jean Paul Riopelle who studied at the Ecole Polytechnique de Montreal from 1939 to 1940.
![CiC_Issue11-example1](https://user-images.githubusercontent.com/57008591/67962877-083d2280-fbfe-11e9-9a4c-a28c4243e71d.png)

But we may need to have a different pattern, in order to link the student with the University or School as a group and not just a physical place.

It could be possible to model the education as being a member of a group, just like if he was a employee of a manufacturer, but a member in the role of a student. With this modeling, the same example of Jean Paul Riopelle would be modeled as following:
![CiC_Issue11-example2](https://user-images.githubusercontent.com/57008591/67963026-3884c100-fbfe-11e9-8d21-7ff9a2db28ec.png)

With this pattern, we will be able to link the person to the group.

### Modeling Professorship and other occupations linked to a group

Should we also model in the same way other occupations like professors, worker in a company? Those kind of occupations linked to a group would benefit from such a pattern, but then we would have two kinds of occupations, the ones that are not directly linked to a group, modeled with the `F51 Pursuit` pattern, and the one linked to a group with the `E85 Joining` and `E86 Leaving` pattern. Such a difference for the same kind of activity is problematic in my opinion.

A solution could be to document all occupations in the `F51 Pursuit` (and `E7 Activity` for students) AND in the same time model the ones linked to a institution or company with the Group belonging pattern. This solution has the major drawback of creating duplicates of the same information.

**Comment by KarineLeonardBrouillet** [comment]
I am not sure I quite understand how a university would be different from a typical employer in the case of professors? Even in the case of students, any employer could also have interns no (in other words paid or unpaid staff that is in training) which would be quite similar conceptually (although I recognize these are vastly different conceptually, but from a modeling standpoint I don't see how they would be represented differently)? To me, the belonging we are representing here is more akin to a relationship of influence where the student is the influenced is the student and the influencer is the University or School group?

**Comment by Habennin** [comment]
It seems it comes down to whether you want to emphasize the membership aspect or the activity aspect. You could model it as both and say that the Joining triggered the F51 Pursuit.

**Comment by stephenhart8** [comment]
Is there no risks of having duplicates of the information if we both have the group membership and the F51 Pursuit, or do you think it's two different kinds of information?

**Comment by Habennin** [comment]
I think there is a risk... basically the Pursuit class acts like a Phase class would work. 

You end up with three ways of modelling the same thing. 

No Time

E21->p107->E74

Time via Events

E21 -> p144i -> E85 -> p4 -> E52

**Comment by illip** [comment]
During the Semantic Committee meeting on 2021-01-07, we have identified two realities that our model has to cover:

1. The fact of being a member of a group: Person X was a student at Y University. We might not know if the student went on the campus or not.
2. The fact of doing something somewhere: Person X attended a conference at Y University. Person X might be a member of Y University or not.

Actually, depending on the input data, the line between 1 and 2 might be thin. In most cases, we think museums record 1 as structured data and 2 in a free text field when the information is available. However, in order to ensure the best semantics, CHIN must probably work on three components:

* A good document to explain the differences between both patterns.
* Develop queries that will encompass both options when someone is asking a general question about schooling
* Work on a crossover between both patterns

We do not know yet if the third bullet is mandatory, this will be discussed at our next Semantic Committee meeting.

**Comment by illip** [comment]
After a discussion with @stephenhart8 here is the beginning of a proposal that CHIN will present to its Semantic Committee on 2021-02-04.

1. Both patterns are relevant and answer different needs. The [Group Belonging](https://chin-rcip.github.io/collections-model/target-model/current/social-bonds#group-belonging) pattern is useful to express membership (e.g. As a student of Y University, Person X is a member of this university) while the [Occupation](https://chin-rcip.github.io/collections-model/target-model/current/life-events#occupation) (`F51_Pursuit`) and [Social Status](https://chin-rcip.github.io/collections-model/target-model/current/life-events#social-status) (`E7_Activity`) patterns are useful to describe the activities (e.g. Person X studied history from 2010 to 2016).

2. Currently, there is no link between those two patterns. Therefore, there is no way to know if the activity was performed as being a member of the group. We cannot compare the time-spans as the person could have performed an activity not related to the group while being a member of this group. We cannot use E53_Place as the activity could be performed remotely.

3. So let's say we have this use case in one of our dataset: "Person X studied remotely at Y University". This statement does not state the membership of Person X but mentions a group related to the activity (e.g. doing a MOOC does not always mean that you are a member of the institution). If we are right, we think our model does not support this kind of information. For the moment, we have identified two options:

a) Create sub-events so that we could state that the "study remotely" activity is a sub-event of the Y University's "Hosting Activities" activity. The pattern could look like this:

![Issue11_2020-01-20-Page-1](https://user-images.githubusercontent.com/48293227/105204677-72e3c200-5b12-11eb-8fb6-951d2deb6f96.png)

CHIN always tried to avoid sub-event patterns as it might bring more complexity to the SPARQL queries as we would always need to look for sub-events.

b) Add the Y University as one of the main participants to the activity using `PC14_carried_out_by`. The role could be something as "Host". The question here is to decide whether or not the group can be described as a main participant since the same pattern does not exist for `P11_had_participant`. The pattern could look like this:

![Issue11_2020-01-20-Page-2](https://user-images.githubusercontent.com/48293227/105204798-83943800-5b12-11eb-9676-a485ffdb7116.png)

Note that both patterns might require the creation of a sub-event pattern in order to state the whole schooling of someone that encompasses all the different institutions where the person has studied.

Four questions:

1. Do we need this new pattern?
2. If so, what would be the best approach?
3. Should we use the [Occupation pattern](https://chin-rcip.github.io/collections-model/target-model/current/life-events#occupation) or the [Social Status](https://chin-rcip.github.io/collections-model/target-model/current/life-events#social-status) pattern to handle schooling?
4. Do we need to include the umbrella activity to encompass all the schooling activities from different institutions?

**Comment by marielmat** [comment]
Concernant nos discussions entourant ce sujet lors de la dernière réunion du comité (2021-02-04), voici les trois exemples d'individus associés au Séminaire de Québec. Ils ont eu plusieurs types d'appartenance et d'affiliation, parfois simultanément. J'aimerais voir comment cela se reflèterait globalement dans le modèle.
Je sais que c'est du texte et je suppose que plusieurs données sont déjà disponibles ailleurs. Mais je pense que l'effort nous aiderait à voir où nous pouvons nous améliorer au moment de consigner des métadonnées sur les E39 Actor.

http://www.biographi.ca/fr/bio/laflamme_joseph_clovis_kemner_13F.html
http://www.biographi.ca/fr/bio/demers_jerome_8F.html
https://collections.mcq.org/objets/297208 (voir Notice biographique)

**Comment by illip** [comment]
Merci @marielmat, nous allons profiter de la rencontre du 2021-03-04 pour tenter une première modélisation de tes exemples. @stephenhart8 et moi allons essayer d'identifier les énoncés biographiques pertinents à modéliser avant la prochaine rencontre notamment ceux concernant la scolarité. Un énorme merci, je crois que ceux-ci permettront de vérifier si notre proposition est pertinente.

During the last Semantic Committee meeting, we have decided to keep this issue open for two reasons:

1. We want to test our patterns with some use cases like the ones presented by @marielmat.
2. We will not implement the institution's participation in our current pattern as there is no ideal solution at the moment.

Since having a P11.1 property would be the easiest way for solving this issue, the committee would like to raise this use case to the CRM SIG to see if it would be possible to add this construct in order to attribute roles to any kind of participants of an event.

The current status should also be documented in the Target Model.

**Comment by illip** [comment]
The Target Model has been updated according to our decisions regarding this issue. Essentially, we have updated three different sections:

1. Occupation pattern
2. Social Status pattern
3. Group Belonging

In each of them, we explain that the Group Belonging pattern would probably the most common one to model schooling, but that the other ones are also good candidates depending on the input data. However, we still don't know which one between Occupation and Social patterns is the best one in the scenario where we document solely the activity and not the institution. This topic will be discussed in Issue #27.

We have also created Issue #66 to discuss the relevance of P11.1 creation.

The only remaining task before closing this issue is to include an introduction to CRMpc.
